distributions/aws/test/e2e/utils/mysql.py

```diff
@@ -0,0 +1,20 @@
+import mysql.connector
+
+
+def query(mysql_client: mysql.connector.MySQLConnection, query):
+    """
+    Query a mysql database using SQL syntax
+    """
+
+    cursor = mysql_client.cursor(dictionary=True)
+
+    cursor.execute(query)
+
```

Comment:
can we wrap this under exception handler, in case if it fails we should know.

Reply:
sure

Reply:
Apparently an exception is raised by default when the query fails

```
    def execute(self, operation, params=None, multi=False):
        """Executes the given operation

        Executes the given operation substituting any markers with
        the given parameters.

        For example, getting all rows where id is 5:
          cursor.execute("SELECT * FROM t1 WHERE id = %s", (5,))

        The multi argument should be set to True when executing multiple
        statements in one operation. If not set and multiple results are
        found, an InterfaceError will be raised.

        If warnings where generated, and connection.get_warnings is True, then
        self._warnings will be a list containing these warnings.

        Returns an iterator when multi is True, otherwise None.
        """

       ...
    
            except errors.Error:
            raise
        except Exception as err:
            raise errors.InterfaceError(
                "Failed executing the operation; %s" % err)
```